projects/addon-doc/src/components/page/page.template.html

```diff
@@ -44,13 +55,6 @@ <h1 class="t-title">
             </ng-template>
         </ng-container>
     </tui-tabs-with-more>
-    <tui-doc-source-code
-        class="t-source-code"
-        [header]="header"
-        [package]="package"
-        [type]="type"
-        [path]="path"
-    ></tui-doc-source-code>
```

Comment:
![diff](https://user-images.githubusercontent.com/35179038/217529297-bd5eaa6b-1c76-41fa-9956-0c823bb47f9a.gif)

Reply:
Most likely it is better to discuss with UI designer in a separate issue. I revert back style changes